courier/handler.go

```diff
@@ -62,10 +70,9 @@ type listCourierMessagesResponse struct {
 
 // Paginated List Courier Message Parameters
 //
-// nolint:deadcode,unused
 // swagger:parameters listCourierMessages
 type ListCourierMessagesParameters struct {
-	migrationpagination.RequestParameters
+	keysetpagination.RequestParameters
```

Comment:
Is this a breaking change? Is this endpoint released already?

Reply:
It is a breaking change, but this endpoint is not released in OS and in the network it wasn't reachable.

So I'd argue this is fine.